changelog/3822.feature.rst

```diff
@@ -0,0 +1 @@
+Integrating `parfive==1.1rc2` with sunpy.
```

Comment:
```suggestion
Add support for parfive 1.1. This sets a limit on the number of open connections to JSOC when downloading files to 10.
```

Reply:
I'll do this along with the change suggested below. I was resolving a merge conflict and now can't commit this suggestion. :man_facepalming: